homeassistant/components/sensibo/coordinator.py

```diff
@@ -72,7 +72,13 @@ async def _async_update_data(self) -> dict[str, dict[str, Any]]:
             )
             if temperatures_list:
                 temperature_step = temperatures_list[1] - temperatures_list[0]
-            features = list(ac_states)
+
+            features_list = list(ac_states)
```

Comment:
Normally the supported features should not change during the entity lifetime. What's the case for this device that requires this to be checked dynamically?

Reply:
Different modes, different features. For example for fan you can't set temperature. 
Have not tested yet so not sure how this behaves. That's why I put it in draft.

Reply:
Hi @MartinHjelmare 
So maybe some advice here then in terms of different options.
1. Change the supported features depending on state. Presents good in the UI but doesn't block the user from executing the service so would still have to put in a guard.
2. Skip the supported features (full set) and just put in the guards in the services. Would then show options in UI for example to change temperature when that is not possible.
3. As number 2 but also then fix the properties to only show/use the ones linked to the used feature currently in use

I feel tempted to go for 1 given it presents itself correctly in the UI but is that ok given what you wrote in your initial comment?

Reply:
If supported features change it will break everything that is reading then at startup which is at least Alexa , google assistant , HomeKit, and others. There are likely other things that will break as well.

Usually we present the supported features as all the features that the device can possibly support.

Reply:
Good remark. Didn't think about that to be honest.
So I would assume the option 2 is the simplest way to deal with this e.g. place guards in each service possibly affected by the featureset from certain modes.

Reply:
2 with raising HomeAssistantError or similar when something can't be done is likely the most compatible path here.